roles/build_project_repo/tasks/main.yml

```diff
@@ -40,7 +40,7 @@
       # register repo_info for group_vars
       register: repo_info
 
-    - name: Get python package version
+    - name: Get the version for the python package/app being deployed
```

Comment:
Do you need to discover this on each machine? Could you define it in vars instead?

Reply:
This is the version of the app that we're deploying, so it is specific to the app+install (which roughly equates to being specific to the machine, at least for our current setup). This is one of the things I was asking about when I joined ansible open house last week, it's a variable we use for paths but we have to run some code to determine it.

Reply:
Ah, so the variable you're establishing is not the python version, it's a deployment timestamp or something similar that changes with every fresh code deploy? If you're using a symlink  (like `current`) you might be able to define the correct path by specifying the symlink and `follow: true` instead of using this variable.

Reply:
thank you for explaining that more clearly! yes, that's exactly the case

in fact, I remember now that I set a playbook-specific variable that uses the `current` symlink for this playbook, since we're not deploying a new version and the version that would be deployed isn't guaranteed to match the current version with the settings we want to update